.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
A comment here about when this can be moved to the `pyproject.toml` would be useful. Could be [a link to this](https://gitlab.com/pycqa/flake8/-/issues/428#note_251982786)